gobblin-runtime/src/main/java/gobblin/runtime/TaskContext.java

```diff
@@ -251,9 +259,10 @@ public TaskPublisher getTaskPublisher(TaskState taskState, TaskLevelPolicyCheckR
    */
   public DataWriterBuilder getDataWriterBuilder(int branches, int index) {
     try {
-      return (DataWriterBuilder) Class.forName(this.workUnit
-          .getProp(ForkOperatorUtils.getPropertyNameForBranch(ConfigurationKeys.WRITER_BUILDER_CLASS, branches, index),
-              ConfigurationKeys.DEFAULT_WRITER_BUILDER_CLASS)).newInstance();
+      String dataWriterBuilderClassName = this.workUnit.getProp(
+          ForkOperatorUtils.getPropertyNameForBranch(ConfigurationKeys.WRITER_BUILDER_CLASS, branches, index),
+          ConfigurationKeys.DEFAULT_WRITER_BUILDER_CLASS);
+      return (DataWriterBuilder) Class.forName(dataWriterBuilderClassName).newInstance();
```

Comment:
Minor comment, but if you want to split the `return` line into multiple lines, then you can move the `String dataWriterBuilderClassName = ...` declaration outside of the try, catch block.